ui/components/app/snaps/snap-ui-renderer/components/heading.ts

```diff
@@ -5,11 +5,22 @@ import {
 } from '../../../../../helpers/constants/design-system';
 import { UIComponentFactory } from './types';
 
+export const generateSize = (size: HeadingElement['props']['size']) => {
+  switch (size) {
+    case 'md':
+      return TextVariant.headingSm;
```

Comment:
Not sure I understand the reasoning for `md` to map to `sm` 😄

Reply:
We wanted to start with `md` and `lg` but I later discovered that the default size right now is `sm` and we need to introduce `lg`

Reply:
I could default to `sm` and add a `md` and `lg` 1:1 mapping or add the `sm` to the snap component props

Reply:
We shouldn't change the default and we can add `sm` to the props at any time, after or now is up to you

Reply:
Updated :)